src/opening-period/page/EditOpeningPeriod.tsx

```diff
@@ -0,0 +1,271 @@
+import React, { useEffect, useState, useCallback } from 'react';
```

Comment:
Comment about file name: Would it be better to include the word 'Page' in the names of these files EditOpeningPeriod.tsx and EditOpeningPeriod.scss. I know that the name 'page' is already present at the path of the component. But many times you only see the component name when you read the code. And it could be easier to read if you know from the name already what the component is about. I think we have it with ResourcePage and CreateNewOpeningPeriodPage as well 🤔

Reply:
Oh, yes, I'll fix that

Reply:
Fixed